content/docs/testing-farm.md

```diff
@@ -83,13 +83,29 @@ in the pull request:
 
 ## Creating Tests
 
-The easiest way to get started is to use the [tmt][tmt] tool
+The easiest way to get started with defining tests is to use the [tmt][tmt] tool
 which will help you with the setup.
 Please follow [Fedora tmt page][fedora-tmt].
 
+This will create a tree structure with tests and plans in your repository:
+```
+$ tmt
```

Comment:
This does not create anything. When `tmt` is used with no arguments it only shows what's around.

Reply:
that was actually my point :) to show an existing, already initialized, project

is that confusing in this context?

Reply:
I changed the section to be more clear.